Assets/Scripts/Core/MapPos.cs

```diff
@@ -0,0 +1,92 @@
+using System.Collections.Generic;
+
+namespace OperationBlackwell.Core {
+	public class MapPos {
+		public int x;
+		public int y;
+		public float offsetX;
+		public float offsetY;
+		public bool straightToOffset = true;
+
+		public MapPos(int _x, int _y, float _offsetX = 0f, float _offsetY = 0f, bool _straightToOffset = true) {
+			x = _x;
+			y = _y;
+			offsetX = _offsetX;
+			offsetY = _offsetY;
+			straightToOffset = _straightToOffset;
+		}
+
+		public bool Equals(MapPos p2) {
+			// Check if this one equals that one
+			return x == p2.x && y == p2.y;
+		}
+
+		public bool EqualsDeep(MapPos p2) {
+			// Check if this one equals that one
+			return x == p2.x && y == p2.y && offsetX == p2.offsetX && offsetY == p2.offsetY && straightToOffset == p2.straightToOffset;
+		}
+
+		public override string ToString() {
+			return "x:" + x + ", y:" + y;
+		}
+
+		public string ToStringThorough() {
+			return "x:" + x + ", y:" + y + "; ox:" + offsetX + ", oy:" + offsetY + ", s: " + straightToOffset;
+		}
+
+		public void ResetOffset() {
+			offsetX = 0f;
+			offsetY = 0f;
+		}
+
+		public MapPos ClearOffset() {
+			return new MapPos(x, y);
+		}
+
+		public MapPos Clone() {
+			return new MapPos(x, y, offsetX, offsetY, straightToOffset);
+		}
+
+		public MapPos AddPosCopy(int x, int y) {
+			return new MapPos(this.x + x, this.y + y);
+		}
+
+		public static int Distance(MapPos p1, MapPos p2) {
+			return System.Math.Abs(p1.x - p2.x) + System.Math.Abs(p1.y - p2.y);
+		}
+		public static bool ListContains(List<MapPos> list, MapPos mapPos) {
+			// Check if map pos is in list
+			foreach (MapPos pos in list)

```

Comment:
Coding style: add "{" and "}" to the foreach statement and the if-statement

Reply:
This is not resolved yet, please fix. Same goes for line 60.